assignments/callbacks.js

```diff
@@ -1,34 +1,68 @@
 const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
+function callBack(output, bool) {
+  console.log(output, bool);
```

Comment:
i would rewrite this console.log to be more descriptive:
`console.log('callBack output: ' + output); \\This is how I know what is displaying in the console`